plugins/blip/basics/less/DashboardSection.less

```diff
@@ -28,6 +28,20 @@
     margin: 0;
     i {
       float: right;
+
+      &.icon--toggle {
+        margin-left: 15px;
+        float: none;
+        color: #bcbec0;
```

Comment:
Kinda. We had it defined within [`CalendarContainer.less`](https://github.com/tidepool-org/tideline/pull/299/commits/b945170d997d9193d337328e38904dce21d40a7f#diff-1ecbd7da6783763477f103a5bff1e7a1L1), so I moved it into [`variables.less`](https://github.com/tidepool-org/tideline/pull/299/commits/b945170d997d9193d337328e38904dce21d40a7f#diff-cc77fdc12c69e0f940a6c1108c789313R21).